components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -3,8 +3,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef BRAVE_COMPONENTS_BRAVE_REWARDS_BROWSER_REWARDS_SERVICE_IMPL_H_
-#define BRAVE_COMPONENTS_BRAVE_REWARDS_BROWSER_REWARDS_SERVICE_IMPL_H_
+#ifndef SRC_BRAVE_COMPONENTS_BRAVE_REWARDS_BROWSER_REWARDS_SERVICE_IMPL_H_
```

Comment:
these are not correct, they should not include `SRC_`

Reply:
👍 fixed but according to cpplint, so I will ignore the below issues in the future:

```Terrys-iMac-Pro:brave-browser terrym$ cpplint src/brave/components/services/bat_ledger/bat_ledger_impl.h
src/brave/components/services/bat_ledger/bat_ledger_impl.h:6:  #ifndef header guard has wrong style, please use: SRC_BRAVE_COMPONENTS_SERVICES_BAT_LEDGER_BAT_LEDGER_IMPL_H_  [build/header_guard] [5]
src/brave/components/services/bat_ledger/bat_ledger_impl.h:177:  #endif line should be "#endif  // SRC_BRAVE_COMPONENTS_SERVICES_BAT_LEDGER_BAT_LEDGER_IMPL_H_"  [build/header_guard] [5]
Done processing src/brave/components/services/bat_ledger/bat_ledger_impl.h
Total errors found: 2```